api/v1beta1/network_disruption.go

```diff
@@ -190,6 +190,126 @@ func (s *NetworkDisruptionSpec) GenerateArgs() []string {
 	return args
 }
 
+// Format describe a NetworkDisruptionSpec
+func (s *NetworkDisruptionSpec) Format() string {
```

Comment:
Is it possible to have unit test to check each behaviors?

Reply:
@aymericDD I created them! Could you take a look?